src/query/api/v1/handler/prometheus/native/read.go

```diff
@@ -145,6 +145,11 @@ func (h *promReadHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	fmt.Println("Exhaustive:", result.Meta.Exhaustive)
+	if !result.Meta.Exhaustive {
+		w.Header().Set(handleroptions.LimitHeader, handleroptions.LimitHeaderSeriesLimitApplied)
```

Comment:
We actually don't need this here for this handler since we do this as part of "handleroptions.AddWarningHeaders(w, result.Meta)" which happens above yeah?